src/app/core/learning-object-module/search/search.service.ts

```diff
@@ -1,9 +1,94 @@
 import { Injectable } from '@angular/core';
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { LearningObject } from '@entity';
+import { throwError } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+import { Query } from 'app/interfaces/query';
+import * as querystring from 'querystring';
+import { SEARCH_ROUTES } from 'app/core/learning-object-module/search/search.routes';
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: 'root',
 })
 export class SearchService {
+  constructor(private http: HttpClient) {}
 
-  constructor() { }
+  /**
+   * Fetches Array of Learning Objects
+   *
+   * @returns {Promise<LearningObject[]>}
+   * @memberof LearningObjectService
+   */
+  getLearningObjects(
```

Comment:
Same thing as the other PRs. If you change the class that the function is in all of the references will need to be updated. VS Code has some shortcuts that make it easy to find where the references are.